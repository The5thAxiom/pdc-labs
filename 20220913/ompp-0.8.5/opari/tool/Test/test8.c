int main() {
  int i;

#pragma omp inst init

#pragma omp inst off

#pragma omp inst begin(foo)

#pragma omp parallel
      i = 9;

#pragma omp inst end(foo)

#pragma omp inst on

#pragma omp inst finalize

}
