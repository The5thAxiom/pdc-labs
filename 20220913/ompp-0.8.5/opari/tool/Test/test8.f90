      program test8

C$OMP INST INIT

      !$OMP INST OFF

      !$OMP INST BEGIN(foo)

      !$OMP PARALLEL
      i = 9
      !$OMP end PARALLEL

      !$OMP INST END(foo)

      !$OMP INST ON

   !$OMP INSTFINALIZE

      end
