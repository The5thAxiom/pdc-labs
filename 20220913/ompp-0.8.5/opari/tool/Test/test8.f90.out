#ifdef _POMP
#  undef _POMP
#endif
#define _POMP 200110

#line 1 "test8.f90"
      program test8

      call POMP_Init()
#line 4 "test8.f90"

      call POMP_Off()
#line 6 "test8.f90"

      call POMP_Begin(29)
#line 8 "test8.f90"

      call POMP_Parallel_fork(30)
#line 9 "test8.f90"
      !$omp parallel
      call POMP_Parallel_begin(30)
#line 10 "test8.f90"
      i = 9
      call POMP_Barrier_enter(30)
!$omp barrier
      call POMP_Barrier_exit(30)
      call POMP_Parallel_end(30)
#line 11 "test8.f90"
      !$omp end parallel
      call POMP_Parallel_join(30)
#line 12 "test8.f90"

      call POMP_End(29)
#line 14 "test8.f90"

      call POMP_On()
#line 16 "test8.f90"

      call POMP_Finalize()
#line 18 "test8.f90"

      end
